src/components/panel/panel.spec.js

```diff
@@ -788,7 +889,6 @@ describe('$mdPanel', function() {
         openPanel(config);
 
         var panelCss = document.querySelector(PANEL_EL).style;
-        expect(panelCss.position).toEqual('fixed');
```

Comment:
I think this needs to be in at least 1 test, since it actually does verify that positioning is correct.